numbersofinterest/NumbersOfInterest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+class NumbersOfInterest
+{
+    public static function calculate($input)
```

Comment:
would investigate turning input into a data structure (e.g. `class Input`) so that parsing is separated from its usage